apps/prairielearn/src/ee/pages/institutionAdminGeneral/institutionAdminGeneral.ts

```diff
@@ -33,6 +38,8 @@ router.get(
     res.send(
       InstitutionAdminGeneral({
         institution,
+        authn_providers,
```

Comment:
I believe this isn't used on the page.

Reply:
Correct. This has now been removed in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6).